src/layers/legacy/medCoreLegacy/database/medDatabaseProxyModel.cpp

```diff
@@ -62,41 +62,47 @@ void medDatabaseProxyModel::clearAllFilters()
 
 bool medDatabaseProxyModel::customFilterAcceptsRow( int source_row, const QModelIndex & source_parent ) const
 {
-
     // get the current model index
     QModelIndex current(sourceModel()->index(source_row, 0, source_parent));
     // get the data we want to check
     QModelIndex index = sourceModel()->index(source_row, currentKey, source_parent);
 
     // show all children if parent is valid
-    if (current.parent().isValid() && !isCheckingChild && !isCheckingParent)
+    if (current.parent().isValid() && (isCheckingChildren == 0))
```

Comment:
!isCheckingChild is better than (isCheckingChildren == 0)

Reply:
isCheckingChildren is not a boolean here. The changes are a bit old, but have been made by @ocommowi  and @fcollot in this commit https://github.com/Inria-Asclepios/medInria-public/commit/676758363b5aec601098c24ae09cce24864a2b41#diff-8a3c89be0d75ff3017f2e8437c78aebb

Reply:
(if you do not have access to this commit, its name is "Enable multiple depth levels instead of two in search" and change only that part)

Reply:
Ok je savais pas
Bizarre un is qui est pas bool...

Reply:
(I merged that PR, but I did not work on it)